service-commands/scripts/setup-instance.sh

```diff
@@ -138,25 +99,26 @@ case "$service" in
 			"[[ ! -d ~/audius-protocol ]]" \
 			"&& set -x" \
 			"&& git clone https://github.com/AudiusProject/audius-protocol.git" \
-			"&& PROTOCOL_DIR=/home/ubuntu/audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref" \
-			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh" || true
-        reboot_instance $provider $name
+			"&& PROTOCOL_DIR=audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref" \
+			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh"
+		reboot_instance $provider $name
 
 		wait_for_instance $provider $user $name
 
 		# perform once more for audius-client
 		execute_with_ssh $provider $user $name \
-			"PROTOCOL_DIR=/home/ubuntu/audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
+			"bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
 
-		execute_with_ssh $provider $user $name "nohup npm run start:dev:cloud > ~/audius-client.out 2>&1 &"
-		execute_with_ssh $provider $user $name "source ~/.nvm/nvm.sh; source ~/.profile; A up || (A down && A up)"
-
-		copy_file_to_remote $provider $user $name "~/.gitconfig" "~/.gitconfig"
+		# start up frontend and backend
+		execute_with_ssh $provider $user $name "set -x; nohup npm run start:dev:cloud > ~/audius-client.out 2>&1 &"
+		execute_with_ssh $provider $user $name "set -x; source ~/.nvm/nvm.sh; source ~/.profile; A up"
 
+		# configure local files: /etc/hosts, /etc/ssh/ssh_config.d/60-audius.conf
 		configure_etc_hosts
```

Comment:
Let's make this optional

Reply:
Which part do you want to be optional?

Since the /etc/hosts bit is always required I figured not to make it optional. Or are there times when we're running multiple remote-dev boxes?

And setting up /etc/ssh/ssh_config.d/60-audius.conf will only happen once if the file doesn't exist. My thought was for new-hires it would create the file, but for anyone else we could simply `touch` it and opt-out.

Reply:
Did both for now.